src/main/java/org/junit/runner/Request.java

```diff
@@ -166,4 +167,15 @@ public Request filterWith(final Description desiredDescription) {
     public Request sortWith(Comparator<Description> comparator) {
         return new SortingRequest(this, comparator);
     }
+    
+    /**
+     * Returns a Request whose Tests can be run in a certain order, defined by
+     * <code>sorter</code>
+     *
+     * @param sorter the sorter determining the order of the tests in this Request
+     * @return a Request with ordered Tests
+     */
+    public Request sortWith(Sorter sorter) {
```

Comment:
I just realized that there is no need for this method, since Sorter implements `Comparator<Description>`